backend/.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
I would also add these:
```
.dockerignore
Dockerfile
.git
.gitignore
```

Reply:
No, this should contain `*` and then very short list of things that are actually required for image to be built instead.
This way nothing extra is included and as soon as something is missing it will stop working. It is also a good approach for self-documentation.